geonode_mapstore_client/client/themes/default/less/geonode.less

```diff
@@ -71,3 +71,8 @@
         width: 98%;
     }
 }
+
+// overrides to fix save / save as modal height
+.ms-resizable-modal > .modal-content {
```

Comment:
Probably this style can be replaced by the ResizeModal props `fitContent` if not this style should be more specific to the save modal to avoid issue with other modals in the app